packages/react-vanilla-components/src/components/Switch.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback } from 'react';
+import { withRuleEngine } from '../utils/withRuleEngine';
+import { PROPS } from '../utils/type';
+import FieldWrapper from './common/FieldWrapper';
+
+const Switch = (props: PROPS) => {
+  const {
+    id,
+    label,
+    enum: enums,
+    enumNames,
+    required,
+    value,
+    name,
+    enabled,
+    visible,
+    appliedCssClassNames
+  } = props;
+
+  const selectedValue = enums?.[0];
+  const handleChange = useCallback(
+    (e: React.ChangeEvent<HTMLInputElement>) => {
+      const val = e.target.checked ? selectedValue : 'undefined';
```

Comment:
Use unSelectedValue like this
const unSelectedValue = enums?.[1] || null;

Reply:
Implemented!